### PR TITLE
Authentication

### PR DESCRIPTION
**Pull request by varun-Tad**
Files/Folders to review

1. AuthSlice.js
2. SignInpage folder
3. SignUpPage folder

credentials :
email: mike@gmail.com
password: 12341234

Preview: https://effortless-pavlova-49805c.netlify.app/

### CODE REVIEW

File: dressvid/src/App.js [hunk 1]
```diff
@@ -10,6 +10,10 @@ import { LikeDislikepage } from "./routes/LikeDislikepage/LikeDislikepage";
 import { Singlepage } from "./routes/Singlepage/Singlepage";
 import Playlistpage from "./routes/PlaylistPage/Playlistpage";
 import { Singleplaylistpage } from "./routes/PlaylistPage/Singleplaylistpage";
+import SignInpage from "./routes/SignInPage/SignInpage";
+import SignUppage from "./routes/SignUpPage/SignUppage";
```
Comment (sri-rishi) [inline dressvid/src/App.js:14]: Suggestion: can import all routes files from index.js in routes folder

File: dressvid/src/components/sidebar/sidebar.js [hunk 2]
```diff
@@ -47,10 +73,17 @@ const Sidebar = () => {
...
-                </Link>
+                {userStatus ? (
+                  <Link to={item.path}>
+                    {item.icon}
+                    <span>{item.title}</span>
+                  </Link>
+                ) : (
+                  <Link to={item.altpath}>
+                    {item.icon}
+                    <span>{item.title}</span>
```
Comment (sri-rishi) [inline dressvid/src/components/sidebar/sidebar.js]: Suggestion: you can destructure item, like {path, title, altpath}

File: dressvid/src/routes/Explorepage/Explorepage.js [hunk 3]
```diff
@@ -29,31 +29,23 @@ import { AiOutlineClockCircle } from "react-icons/ai";
...
+import { toast } from "react-toastify";
+import "./Explorepage.css";
 
 const Explorepage = () => {
   const word = "selectedExploreOptions";
   const exploreSelected = useSelector((state) => state.explore[word]);
-
   const watchStatusSelected = useSelector(
     (state) => state.statusLater.watchedLaterNums
   );
```
Comment (sri-rishi) [inline dressvid/src/routes/Explorepage/Explorepage.js:48]: Suggestion: you can use optional chaining (?) to check that each reference in the chain is valid.

File: dressvid/src/components/sidebar/sidebar.js [hunk 4]
```diff
@@ -1,19 +1,39 @@
 import React, { useState } from "react";
-
 import * as FaIcons from "react-icons/fa";
 import * as AiIcons from "react-icons/ai";
 import { BsDisplay } from "react-icons/bs";
 import { FaRegUserCircle } from "react-icons/fa";
-import { Link } from "react-router-dom";
```
Comment (sri-rishi) [inline dressvid/src/components/sidebar/sidebar.js:7]: suggestion: you can import all icons in icons.js and export from there

**Review by sri-rishi — APPROVED** [review]
code and ui looks good, just some suggestions which I think can be done else feel free to merge

File: dressvid/src/components/sidebar/sidebar.css [hunk 5]
```diff
@@ -106,7 +112,7 @@
 
 .sideBar-text a {
   text-decoration: none;
-  color: #f5f5f5;
+  color: var(--color-whiteSmoke);
```
Comment (OmKakatkar) [inline dressvid/src/components/sidebar/sidebar.css:115]: Great to use of color variables

**Review by OmKakatkar — APPROVED** [review]
Functionality is working as expected

**Review by Divya70 — APPROVED** [review]
looks good to me , you can merge

**Pull request merged**